src/components/sidebarContent/TestData.vue

```diff
@@ -16,7 +16,7 @@
         :key="list.name"
         :text="list.name"
         style-type="solid"
-        @click="_addList(JSON.parse(JSON.stringify(list)))"
+        @click="_addTestList(JSON.parse(JSON.stringify(list)))"
```

Comment:
I think it would be better to move JSON.parse(JSON.stringify( into the _addTestList.
That we have more clear html. And still we have function for that - we can move as much js as we can there